x-pack/plugins/siem/public/resolver/view/process_event_dot.tsx

```diff
@@ -578,7 +578,7 @@ export const ProcessEventDot = styled(ProcessEventDotComponents)`
   }
 
   & .related-dropdown {
-    width: 4.5em;
+    width: 8em;
```

Comment:
So there is enough room at the smallest scale to show the common event types without `...` ellipses.